.golangci.yml

```diff
@@ -0,0 +1,14 @@
+run:
+  timeout: 5m
+linters:
+  enable:
+    - gosimple
```

Comment:
Hmm, the docs say that this `gosimple` plus others are enabled by default. Part of that list includes [errcheck](https://github.com/kisielk/errcheck). But when I run `errcheck -verbose ./...` locally, i get

```
go-tfe % errcheck -verbose ./...
checking github.com/hashicorp/go-tfe
checking github.com/hashicorp/go-tfe/examples/workspaces
checking github.com/hashicorp/go-tfe
checking github.com/hashicorp/go-tfe.test
checking github.com/hashicorp/go-tfe/examples/organizations
ip_ranges.go:83:23:     (io.Closer).Close       defer resp.Body.Close()
logreader.go:76:23:     (io.Closer).Close       defer resp.Body.Close()
tfe.go:410:17:  (io.Closer).Close       resp.Body.Close()
tfe.go:594:23:  (io.Closer).Close       defer resp.Body.Close()
```

It doesn't seem like these are enabled by default then? I think we should add all those that are supposed to be enabled by default explicitly here. So
```
linters:
  enable:
   - deadcode
   - errcheck
   - gosimple
   - govet
   - ineffassign
   - staticcheck
   - structcheck
   - typecheck
   - unused
   - varcheck
```